database/migrations/2024_03_18_164714_add_decline_msg_to_checkout_acceptance_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeclineMsgToCheckoutAcceptanceTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('checkout_acceptances', function (Blueprint $table) {
+            $table->string('declined_msg')->after('signature_filename');
```

Comment:
This should be nullable (`->nullable()`).

Reply:
I'm not sure but we might want to make this a `text` instead of a `string` so that users don't type out a paragraph and then get an exception thrown when the length is longer than the default `string`...